fixed-hash/src/hash.rs

```diff
@@ -506,11 +506,7 @@ macro_rules! construct_hash {
 			}
 		}
 
-		#[cfg(all(
-			feature = "heapsize-support",
-			feature = "libc",
-			not(target_os = "unknown")
-		))]
+		#[cfg(feature = "heapsize-support")]
```

Comment:
The `target_os` need to be there I'm afraid. WASM can't use the `heapsize` crate.

Reply:
okay will revert it

Reply:
done